modules/rosa-creating-cluster.adoc

```diff
@@ -20,6 +20,21 @@ link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Share
 
 .Procedure
 
+. Enable ROSA to create JSON files for account-wide roles and policies, including Operator policies. This is supported for the `rosa sts-init --json` mode only:
++
+[source,terminal]
+----
+$ rosa create account-roles --json --version-4.7
```

Comment:
`--json` is not a valid flag. This was changed in the code to `--mode manual` in the actual code change

Valid options for `--mode` are:
- `auto`: Roles and policies will be created using the current AWS account
- `manual`: Policy documents will be saved in the current directory
(the default is `auto`)